### PR TITLE
Added warning about configuration difference from MetalLB docs

### LINKED ISSUE #9973: [stable/metallb] default configmap name is different than documenation
**Is this a request for help?**:
No
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
Bug Report

**Version of Helm and Kubernetes**:
```
helm version
Client: &version.Version{SemVer:"v2.12.0", GitCommit:"d325d2a9c179b33af1a024cdb5a4472b6288016a", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.12.0", GitCommit:"d325d2a9c179b33af1a024cdb5a4472b6288016a", GitTreeState:"clean"}
```
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.3", GitCommit:"435f92c719f279a3a67808c80521ea17d5715c66", GitTreeState:"clean", BuildDate:"2018-11-26T12:46:57Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.3", GitCommit:"435f92c719f279a3a67808c80521ea17d5715c66", GitTreeState:"clean", BuildDate:"2018-11-26T12:46:57Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
stable/metallb-0.8.1

**What happened**:
In the helm chart the default name value for the config map is `metallb-config` (via `existingConfigMap` setting)

In the official documentation it states to create a map called `config` under the same namespace.

**What you expected to happen**:
The default names be aligned between documentation, helm chart installation, and manual `kubectl apply` installation

**How to reproduce it** (as minimally and precisely as possible):
Follow the documentation:

1. Install metallb via helm `helm install --name metallb stable/metallb`
2. Create config map: https://metallb.universe.tf/configuration/
  (can use sample config for different name: https://raw.githubusercontent.com/google/metallb/v0.7.3/manifests/example-config.yaml)

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by ebauman**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed

### CODE REVIEW

File: stable/metallb/README.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ By default, this chart does not install a configuration for MetalLB, and simply
 warns you that you must follow [the configuration instructions on MetalLB's
 website][metallb-config] to create an appropriate ConfigMap.
 
+**Please note:** By default, this chart expects a ConfigMap named 'metallb-config' within the same namespace as the chart is deployed. _This is different than the MetalLB documentation_, which asks you to create a ConfigMap in the `metallb-config` namespace, with the name of 'config'.
+
```
Comment (mattfarina) [inline stable/metallb/README.md:91]: If I look at the docs, linked in the issue, the namespae is `metallb-system` not `metallb-config`. Can you please update or explain the difference?

Comment (ebauman) [reply stable/metallb/README.md:91]: Updated - that was a mistake on my part. See recent commit

**Review by mattfarina — APPROVED** [review]

**Pull request merged**